src/game.cpp

```diff
@@ -10848,6 +10848,10 @@ void game::butcher()
         add_msg(m_info, _("You can't butcher while driving!"));
         return;
     }
+    if( !m.can_put_items_ter_furn( u.pos() ) ) {
+        add_msg( m_info, _( "You can't butcher while standing here!" ) );
```

Comment:
According to the above I would indeed prefer solving issues to fullest extent. Not being able to access a visible item sounds very different from not even knowing there is an item(s) there.